code/src/components/TodoList.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-return-assign */
+/* eslint-disable linebreak-style */
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+
+import todos from '../reducers/todos'
+import './todoList.css'
+
+const TodoList = () => {
+  const items = useSelector((store) => store.todos.items)
+
+  let count = 0
+  const dispatch = useDispatch()
+
+  const onClickAllDone = () => {
+    dispatch(todos.actions.allDone())
+  }
+
+  return (
+    <div>
+      {items.map((todo) => (
+        <div key={todo.id} className="todo-items">
+          <div className="todo-item">
+            <input
+              className="checkbox"
+              type="checkbox"
+              checked={todo.isComplete}
+
```

Comment:
FIGHT WITH LAZYNESS!! 🔫 add REQUIRED to make your to-do free from empty tasks!! 🎖️ 😆